config/Dockerfiles/rpmbuild/Dockerfile

```diff
@@ -41,4 +41,5 @@ COPY rpmbuild-test.sh /home/rpmbuild-test.sh
 # Run the build script
 ENTRYPOINT ["bash", "/home/rpmbuild-test.sh"]
 # Run the container as follows
+# Note: For stage builds, don't do production=no, simply don't pass a production var in
```

Comment:
I guess the comment "production=no" threw me off...since it will be production even though you said no :)

But I think in practice, we are going to have a job parameter that will be production_mode. And in our Jenkinsfile we will startup the script with:

`production=${production_mode}`

The way the code is now...we will need an extra if statement to determine if we need to pass in the env var.